### ISSUE TITLE
[Suggestion] Options to sync terminal theme to current Atom theme

### ISSUE THREAD

**Issue by yhjor1212** [jeremyramin/terminal-plus#263]
To be honest I don't feel comfortable with all the theme provided by terminal-plus. It is not customisable at all (even you change the stylesheet). It would be better if there is such option available where we can just simply extend the background and text color in my current Atom theme.

**Comment by flybirdx101** [comment]
Agreed. This is probably the only thing that makes working with terminal-plus uncomfortable. My Macs terminal uses adapted Dark-One, and using terminal-plus really hits the eyes.

**Comment by The-Penultimate-Defenestrator** [comment]
YES! PLEASE!

**Comment by xjhuang2015** [comment]
Me too.
